backend/grants/admin.py

```diff
@@ -533,9 +533,7 @@ class GrantAdmin(ExportMixin, ConferencePermissionMixin, admin.ModelAdmin):
         ),
     )
 
-    @admin.display(
-        description="User",
-    )
+    @admin.display(description="User", ordering="user__full_name")
```

Comment:
In the admin I cannot order by `user.display_name` unfortunately nor I can do `user__full_name,user__name` as order option, so I choose to order by full_name... 
The default sorting is the primary key btw

Reply:
had no idea this existed, does it mean that when you click on "User" it will sort on full_name? that's cool

Reply:
you can also set the default ordering to be honest, e..g: order by conference and name

so they are grouped together